include/sdbus-c++/Message.h

```diff
@@ -131,6 +131,7 @@ namespace sdbus {
         std::string getInterfaceName() const;
         std::string getMemberName() const;
         std::string getSender() const;
+        std::string getPath() const;
```

Comment:
While you're here, it might be nice to also add:
```c++
std::string getDestination() const;
```

Reply:
@m-blaha Would you be that kind to also add `getDestionation()`? Or shall we rather create a separate PR for that?

Reply:
Of course.
Also I added handling the situation when the message does not have some fields (interface, member, path and destination) set. In this situation the getters from underlying sdbus library return NULL.

Reply:
Or would it be better to just return empty string in such situations? I do not like getters throwing an exceptions.

Reply:
@m-blaha I agree. If it were a true erroneous condition that a sd-bus message wouldn't have any of those fields, then I guess sd-bus API would return an error code. But it just returns `const char*`, where `nullptr` means no data. It seems to be a kind of possible, valid value. So -- I agree -- in our library, we could represent that sentinel value by an empty string, as @dleeds-cpi suggested, rather than throwing exceptions which in sdbus-c++ are used for rather serious, unexpected situations.